primefaces/src/main/java/org/primefaces/component/tabview/TabViewRenderer.java

```diff
@@ -253,25 +253,34 @@ protected boolean encodeTabHeader(FacesContext context, TabView tabView, Tab tab
         //header container
         writer.startElement("li", tab);
         writer.writeAttribute("class", styleClass, null);
-        writer.writeAttribute("role", "tab", null);
-        writer.writeAttribute(HTML.ARIA_EXPANDED, String.valueOf(active), null);
-        writer.writeAttribute(HTML.ARIA_SELECTED, String.valueOf(active), null);
-        writer.writeAttribute(HTML.ARIA_LABEL, tab.getAriaLabel(), null);
-        writer.writeAttribute("data-index", index, null);
+        writer.writeAttribute("role", "presentation", null);
         if (tab.getTitleStyle() != null) {
             writer.writeAttribute("style", tab.getTitleStyle(), null);
         }
         if (tab.getTitletip() != null) {
             writer.writeAttribute("title", tab.getTitletip(), null);
         }
-        if (tabindex != null) {
-            writer.writeAttribute("tabindex", tabindex, null);
-        }
 
         //title
         writer.startElement("a", null);
         writer.writeAttribute("href", "#" + tab.getClientId(context), null);
```

Comment:
Extract `tabGetClientId(context)` into a variabl `String id = tabGetClientId(context)` so you can re-use it.